docs/src/pages/index.tsx

```diff
@@ -1,41 +1,169 @@
-import React from 'react';
-import clsx from 'clsx';
-import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import React, { CSSProperties, ReactNode, useEffect } from 'react';
 import Layout from '@theme/Layout';
-import HomepageFeatures from '@site/src/components/HomepageFeatures';
-
-import styles from './index.module.css';
 
-function HomepageHeader() {
-  const {siteConfig} = useDocusaurusContext();
-  return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
-      <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
-          <Link
-            className="button button--secondary button--lg"
-            to="/docs/intro">
-            Docusaurus Tutorial - 5min ⏱️
-          </Link>
-        </div>
-      </div>
-    </header>
-  );
-}
+import styles from './index.module.scss';
+import clsx from 'clsx';
+import { FaRocket, FaTruck } from 'react-icons/fa';
+import Link, { NavLinkProps } from '@docusaurus/Link';
 
 export default function Home(): JSX.Element {
-  const {siteConfig} = useDocusaurusContext();
+  useEffect(() => {
+    document.body.style.background = 'linear-gradient(90deg,var(--generic-bg) calc(25px - 1px),transparent 1%) 50% / 25px 25px,linear-gradient(var(--generic-bg) calc(25px - 1px),transparent 1%) 50% /25px 25px,var(--generic-color)';
+    return () => {
+      document.body.style.background = '';
+    }
+  }, []);
+
   return (
     <Layout
-      title={`Hello from ${siteConfig.title}`}
-      description="Description will go into a meta tag in <head />">
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
+      // TODO: change title and description?
+      title={`Kargo - GitOps Promotion Tool`}
+      description="Stage to stage GitOps Promotion Made Easy With Kargo">
+      <main className='landing-page'>
+        <header className={styles.header}>
+          <h1 style={{margin: 0}}>Kargo</h1>
+          <span>Learn how to use Kargo for GitOps promotions of stages</span>
+          <div style={{marginTop: '24px'}}>
+            <Link to='/new-docs/user-guide/core-concepts'>
+              <button className='btn'>Overview</button>
+            </Link>
+            <Link to='/new-docs/user-guide/examples'>
+              <button className='btn btn-primary' style={{marginLeft: '24px'}}>Learn By Examples</button>
+            </Link>
+          </div>
+        </header>
+        <div className='container'>
+          <Section 
+            title={<><FaRocket /> Get Started</>}
+            nodes={[
```

Comment:
Can't we do just the regular jsx node instead of the array? The data are not dynamic and it looks better in the code.